README.md

```diff
@@ -1 +1,22 @@
-# hw01
\ No newline at end of file
+##### HOMEWORK ASSIGNMENT 1
+I'm **Jiayang He**, a MAPSS student concentrating on *psychology*.
+
+My interest in the area focuses on several specific topics under certain sub-fields of psychology:
+
+* Psychophysiology
+  + Will stress be contagious via watching stressful videos? 
+* Social neuronscience
+  + How social stress impact interoception?
+* Clinical psychology
+  + Will people with depresssion or panic disorder have a different pattern of stress response?
+
+I am currently working in Dr. Norman's SPNL (Social Psychophysiology & Neuroendocrinology Lab). Here's our website: <https://sites.google.com/a/uchicago.edu/spnl/>.
+
+Besides schoolwork, recently I have had a lot of fun playing a game called *__Animal Crossing__*, and I would totally recommend this game to you. 
+![Image of Animal Crossing](https://s1.gaming-cdn.com/images/products/4809/orig/animal-crossing-new-horizons-switch-cover.jpg)
```

Comment:
Great usage of inserting a picture here! I am wondering whether it would be possible to modify the height and width of the picture, though, since it looks a bit big compared to the texts.

Reply:
@xumengyan01 not easily achieved in Markdown. Would need to manually edit the file in advance.

More so, I recommend a blank line between the image and the previous text so they don't end up mashed together.

```
Besides schoolwork, recently I have had a lot of fun playing a game called *__Animal Crossing__*, and I would totally recommend this game to you. 

![Image of Animal Crossing](https://s1.gaming-cdn.com/images/products/4809/orig/animal-crossing-new-horizons-switch-cover.jpg)
```

Besides schoolwork, recently I have had a lot of fun playing a game called *__Animal Crossing__*, and I would totally recommend this game to you. 

![Image of Animal Crossing](https://s1.gaming-cdn.com/images/products/4809/orig/animal-crossing-new-horizons-switch-cover.jpg)